docs/content/faq/general.md

```diff
@@ -50,15 +50,33 @@ For example, it could answer questions like :
 Data Engineering teams, Data Lake Admins and Operational Analysts would be able to 
 manage and use this feature for operational insights on the Delta Lake. 
 
+**Q Can I run this solution on non-Databricks environment ?**
```

Comment:
The answer must be "no", because of Databricks License this project is distributed by.

Reply:
@nfx , ok. Made the changes